.editorconfig

```diff
@@ -0,0 +1 @@
+insert_final_newline = true
```

Comment:
The line in the editor config, which is about inserting final newlines, it itself missing a final newline.

...the irony, sir, is *staggering*

Reply:
🤔 Can this be done? Did I do it wrong? 🤔

Reply:
Looks like users need to have the editorconfig plugin enabled https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig

This makes this PR, only useful in certain situations and won't help everyone

Reply:
Looks like most people will need a plugin - but also gitlab and github might pick this up on their own.. so.. Yeah, let's go :)

Reply:
Also added a .vscode/settings.json file for the vscode users who _dont_ have the editor config plugin enabled 
This should cover most of the vanilla setups. 

```
{
    "files.insertFinalNewline": true
}

```

Reply:
LGTM, I'll rebase and merge